TTHAnalysis/macros/submit.py

```diff
@@ -0,0 +1,230 @@
+import os
+
+## this script creates a sub-directory in the tree path for a given module
+## and runs the friend tree producer for this module
+
+batch = True
+queue = "batch" #"batch" for Oviedo, "8nh" or others for lxplus
+path  = "/mnt_pool/fanae105/user/sscruz/TTH/samples"
```

Comment:
I would definitely keep this script out of the repo (personally, I never add it to my commits), since it works by hardcoding your path (user specific), friend tree module (analysis specific) and accepted/excluded samples (both), all of which one changes again and again when running. It's only an unnecessary source of changes in our commit history.